app/livechat/imports/server/rest/sms.js

```diff
@@ -1,11 +1,26 @@
+import { HTTP } from 'meteor/http';
 import { Meteor } from 'meteor/meteor';
 import { Random } from 'meteor/random';
 
+import { FileUpload } from '../../../../file-upload';
 import { LivechatRooms, LivechatVisitors, LivechatDepartment } from '../../../../models';
 import { API } from '../../../../api/server';
 import { SMS } from '../../../../sms';
 import { Livechat } from '../../../server/lib/Livechat';
 
+const fileStore = FileUpload.getStore('Uploads');
+
+const getUploadFile = async (details, fileUrl) => {
```

Comment:
looks like this function doesn't need to be `async` anymore, there is no `async` calls inside of it.